azure-stack/hci/manage/register-windows-admin-center.md

```diff
@@ -16,6 +16,9 @@ To use Azure services with Windows Admin Center, you must first [install Windows
    > [!IMPORTANT]
    > In order to use Windows Admin Center to register Azure Stack HCI clusters, your Windows Admin Center gateway must be registered to an Azure Active Directory (Azure AD) application ID that is approved by your organization's Azure AD admin. Register Windows Admin Center on the same management PC you plan to use to register your cluster(s), using the same Azure Active Directory (tenant) ID.
 
+   > [NOTE]
+   > If your network is confogured to use a proxy server for internet connection, it is required to configire the proxy settings on Windows Admin Center Gateway Settings. After configuring the proxy restart ServerManagmentGateway (Windows Admin Center) service on the server installed with Windows Admin Center before proceeding with Azure AD registration, else the registration may fail.
+
```

Comment:
Please fix typo "configire"

Reply:
Fixed the typo and made some changes for ease of understanding.

Reply:
The format of the note section is incorrect -- it needs to be [!NOTE].

Reply:
Please check its corrected now.